tests/unit_tests/test_camera_efficiency.py

```diff
@@ -35,21 +36,17 @@ def camera_efficiency_sst(telescope_model_sst, simtel_path):
 
 
 @pytest.fixture
-def results_file(db, io_handler):
+def results_file(io_handler):
     test_file_name = (
+        "tests/resources/"
         "camera-efficiency-table-North-LSTN-01-za020deg_azm000deg_validate_camera_efficiency.ecsv"
     )
     output_directory = io_handler.get_output_directory(
         label="validate_camera_efficiency",
         sub_dir="camera-efficiency",
         dir_type="test",
     )
-    db.export_file_db(
-        db_name="test-data",
-        dest=output_directory,
-        file_name=test_file_name,
-    )
-
+    shutil.copy(test_file_name, output_directory)
```

Comment:
Probably not needed here, but copy2 also preserves metadata.

Reply:
OK - here not needed, as we just need to read the file and metadata like permissions are not important.

(not note the 'red warning' in the following URL that even copy2 does not preserve all metadata: https://docs.python.org/3/library/shutil.html)